tools/experiments/experiments.html

```diff
@@ -218,6 +245,20 @@ <h2>Release channels</h2>
 
   <section class="content">
     <table id="channels-table" border=1></table>
+    <details id="rtv-details">
+      <summary>Advanced</summary>
+      <p>
+        You can also opt-in to any RTV manually using this form.
+      </p>
+      <p>
+        <em>Warning:</em> This is an advanced setting. It should not be used by AMP publishers, rather it is intended
+        for debugging the AMP runtime itself. To opt-out clear the RTV in the input field.
+      </p>
+      <div>
+        <input id="rtv" placeholder="E.g., 011912312359590" maxlength="15" type="text">
+        <button id="rtv-submit" disabled>opt-in</button>
```

Comment:
Style button to be consistent with other buttons

Reply:
Done